lib/workers/repository/dependency-dashboard.ts

```diff
@@ -178,6 +199,7 @@ export async function ensureDependencyDashboard(
   config: SelectAllConfig,
   allBranches: BranchConfig[],
   packageFiles: Record<string, PackageFile[]> = {},
+  configMigrationRes?: ConfigMigrationResult,
```

Comment:
Now it's never nullable:

```suggestion
  configMigrationRes: ConfigMigrationResult,
```